app/grandchallenge/workstations/static/workstations/js/session-control.mjs

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+function openWorkstationSession(element) {
+    return (event) => {
+        const windowIdentifier = element.dataset.workstationWindowIdentifier;
+        const url = element.dataset.createSessionUrl;
+        const query = element.dataset.workstationQuery;
```

Comment:
`url` and `query` don't appear to be used separately. Why not pass through the `creationURI` instead?